schemas/manifest-schema.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "PiWebImportPluginManifestSchema",
+  "title": "PiWeb import plug-in manifest schema",
+  "description": "Describes a PiWeb import plug-in manifest.",
+  "type": "object",
+  "required": [
+    "id",
+    "title",
+    "description",
+    "provides"
+  ],
+  "properties": {
```

Comment:
Should we set [additionalProperties: false](https://json-schema.org/understanding-json-schema/reference/object#additionalproperties) to disallow arbitrary properties in the manifest?

Currently it is possible to _accidentally_ type `"autor": "actually it's spelled author"` and won't see any squiggly lines.

_This would be needed at several places though._

Reply:
Done.